Fixed strict option not working (#208)
apache/openwhisk-package-alarms: README.md

  - If it's true, the Trigger will fire at the top of the hour/minute (\**:**:00).
  - Otherwise, the Trigger will fire after the specific seconds (\**:**:00-59).
  - If you do not set this value, it is set to the default chosen by the operator.

>   - Both `"true"` and `"false"` are recognized internally as boolean values.

Reviewer: How about this?
```suggestion
  - Optionally, string values, `"true"` and `"false"` are also recognized as boolean values.
```
